### PR TITLE
yelp fragment refactor

### PR DESCRIPTION
**Pull request by aleqi200**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -31,6 +31,7 @@ dependencies {
     compile 'org.apache.httpcomponents:httpcore:4.3.2'
     compile 'commons-io:commons-io:2.4'
 
+    //Loading Dialog
```
Comment (vgudla) [inline app/build.gradle]: Probably need to rebase

File: app/src/main/java/com/grelp/grelp/data/GrouponClient.java [hunk 2]
```diff
@@ -29,7 +29,7 @@
     private static final String DEALS_SEARCH_CONTEXT = "mobile_local";
     private static final String GAPI_CLIENT_ID = "44aa9e633a98fd614f0e7f495e3e6442";
     private static final String CONSUMER_ID = "5b2a214e-5c83-11e3-bbd1-0025906127f6";
-    private static final String MAX_RESULTS = "50";
+    private static final String MAX_RESULTS = "20";
```
Comment (vgudla) [inline app/src/main/java/com/grelp/grelp/data/GrouponClient.java:32]: Is 20 too little ?
Or do you have infinite scrolling ?

File: app/src/main/java/com/grelp/grelp/fragments/YelpDetailFragmentMinimum.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+            };
+            tvOpenYelp.setOnClickListener(openReviewListener);
+            Picasso.with(getContext()).load(yelp.getRatingImgUrl()).into(ratingImg);
+            Resources res = getResources();
+            String reviewNumber = res.getString(R.string.num_reviews_parenthesis, yelp.getReviewCount());
+            numberOfReviews.setText(reviewNumber);
+            category.setText(StringUtil.join(", ", yelp.getCategories()));
+            if (yelp.getYelpReviewList() != null && !yelp.getYelpReviewList().isEmpty()) {
+                Log.d("yelp review count", String.valueOf(yelp.getYelpReviewList().size()));
+                YelpReview review = yelp.getYelpReviewList().get(0);
```
Comment (vgudla) [inline app/src/main/java/com/grelp/grelp/fragments/YelpDetailFragmentMinimum.java:73]: Could this be empty and should you add a check (for deals like online training courses etc) ?

File: app/src/main/java/com/grelp/grelp/util/ApplicationUtil.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
+package com.grelp.grelp.util;
+
+import android.content.pm.PackageManager;
+
+/**
+ * Created by acampelo on 12/6/15.
```
Comment (vgudla) [inline app/src/main/java/com/grelp/grelp/util/ApplicationUtil.java]: Remove boilerplate

**Comment by vgudla** [comment]
Fix minor comments and merge

File: app/src/main/java/com/grelp/grelp/data/GrouponClient.java (hunk 2 above)
Comment (aleqi200) [inline app/src/main/java/com/grelp/grelp/data/GrouponClient.java:32]: 20 is the same as the android and iphone groupon apps

File: app/src/main/java/com/grelp/grelp/util/ApplicationUtil.java (hunk 4 above)
Comment (aleqi200) [inline app/src/main/java/com/grelp/grelp/util/ApplicationUtil.java]: argh, gonna remore

File: app/src/main/java/com/grelp/grelp/data/GrouponClient.java (hunk 2 above)
Comment (vgudla) [inline app/src/main/java/com/grelp/grelp/data/GrouponClient.java:32]: sounds good to me but do you still have infinite scrolling ?

File: app/src/main/java/com/grelp/grelp/fragments/YelpDetailFragmentMinimum.java (hunk 3 above)
Comment (aleqi200) [inline app/src/main/java/com/grelp/grelp/fragments/YelpDetailFragmentMinimum.java:73]: there is a check for empty above line 71

Comment (vgudla) [inline app/src/main/java/com/grelp/grelp/fragments/YelpDetailFragmentMinimum.java:73]: yeah missed it

File: app/src/main/java/com/grelp/grelp/data/GrouponClient.java (hunk 2 above)
Comment (aleqi200) [inline app/src/main/java/com/grelp/grelp/data/GrouponClient.java:32]: yes it still shows infinite scrolling

**Pull request merged**